### DISCUSSION TITLE
Cross-Language Integration with Event Store

### DISCUSSION THREAD

**Discussion by b-barry** [spatie/laravel-event-sourcing#481] [Q&A] [answered]
I'm exploring ways to integrate spatie/laravel-event-sourcing with systems written in other languages. Specifically:

Is there a way to listen to event store changes from outside the Laravel ecosystem?
Alternatively, can we pull events using an offset, similar to how message queues work?
What's the best approach for this?

The goal is to create projections in another language.

**Comment by inmanturbo** [comment, marked as answer]
The event store is just a table in a relational database. You would have to poll the table over tcp with your language's mysql client, or implement your own method. The events can queued for instance so you could implement a queue driver for something like rabbitmq. Or use http and set up an API or implement web hooks.
